docs/development/development-environment-docker/README.md

```diff
@@ -12,114 +12,159 @@ And nothing else!
+The logs can still be accessed like this.
+
+```bash
+# Print the logs of the `frontend` service until the time you execute this command
+docker compose logs frontend
 
+# Print the logs of the `backend` service and follow the log outputs
+docker compose logs -f backend
 ```
-bin/compose up worker
+
+Those commands only start the frontend and backend containers and their dependencies. This excludes the workers, which
+are needed to execute certain background actions. Nevertheless, for most interactions the worker jobs are not needed. If
+needed, the workers can be started with the following command. Be aware that this process will consume a lot of the
+system's resources.
+
+```bash
+# Start the worker and let them run continuously
+docker compose up -d worker
+
+# Start the worker to work off all delayed jobs and shut it down afterwards
+docker compose run --rm worker rake jobs:workoff
 ```
 
 The testing containers are excluded as well, while they are harmless to start, but take up system resources again and
-clog your logs while running. The delayed_job background worker reloads the application for every job in development mode. This is a know issue and documented here: https://github.com/collectiveidea/delayed_job/issues/823
+clog your logs while running. The delayed_job background worker reloads the application for every job in development
+mode. This is a know issue and documented here: https://github.com/collectiveidea/delayed_job/issues/823
 
 This process can take quite a long time on the first run where all gems are installed for the first time. However, these
 are cached in a docker volume. Meaning that from the 2nd run onwards it will start a lot quicker.
 
-Wait until you see `frontend_1 | : Compiled successfully.`
-and `backend_1 | => Rails 6.0.2.2 application starting in development http://0.0.0.0:3000`
-in the logs. This means both frontend and backend have come up successfully. You can now access OpenProject
+Wait until you see `✔ Compiled successfully.` in the frontend logs and the success message from Puma in the backend
+logs. This means both frontend and backend have come up successfully. You can now access OpenProject
 under http://localhost:3000, and via the live-reloaded under http://localhost:4200.
 
 Again the first request to the server can take some time too. But subsequent requests will be a lot faster.
 
 Changes you make to the code will be picked up automatically. No need to restart the containers.
 
-## Docker
+### Storybook
 
-You can stop the processes via Ctrl + C. You can also run everything in the background by adding the `-d` option as
-in `bin/compose up -d`. In that case you'll still be able to see the logs using `docker logs` with the respective
-container name. You can see the started containers using `docker ps`.
+There is a service to launch the storybook of the SPOT design system in the local development environment. To run it,
+simply use:
+
+```bash
+# Start the worker and let them run continuously
```

Comment:
@Kharonus I stumbled upon this comment. It is talking about starting a worker (singular) and letting them (plural) run continuously. The inconsistency of singular and plural made me think: What worker is this comment actually talking about? So if the comment is helpful and we decide that it is good to have it, could we be more specific?